test/test_helper.rb

```diff
@@ -5,11 +5,17 @@
 require 'ar_transaction_changes'
 require 'minitest/autorun'
 
+MiniTest::Test = MiniTest::Unit::TestCase unless defined?(MiniTest::Test)
+
 test_dir = Pathname.new(File.dirname(__FILE__))
 config_filename = test_dir.join("database.yml").exist? ? "database.yml" : "database.yml.default"
 database_yml = YAML.load(test_dir.join(config_filename).read)
 ActiveRecord::Base.establish_connection database_yml['test']
 
+if ActiveRecord::VERSION::MAJOR == 4 && ActiveRecord::VERSION::MINOR >= 2
```

Comment:
can we do `Gem::Version.new(ActiveRecord::VERSION::STRING) > Gem::Version.new("4.2.0")` instead?